features/step_definitions/web_steps.rb

```diff
@@ -1,3 +1,3 @@
 When(/^I click the (.*) "(.*)"$/) do |elem, text|
-  find(elem).find('*', text: text).click
-end
\ No newline at end of file
+  all(elem, text: text, count: 1)[0].click
```

Comment:
I think find(elem, text:text) would be functionally the same as all() with count set to 1, but I will change it since find looks cleaner